mangadex_downloader/upscale_ncnn.py

```
@@ -0,0 +1,261 @@
+
+            cmd = [
+                str(self.binary_path),
+                "-i", input_path,
+                "-o", input_path,
+                "-n", self._model_name,
+                "-s", str(self.scale),
+                "-m", str(self.models_dir),
+                "-f", output_format
+            ]
+
+            result = subprocess.run(
```

Ruff (0.13.2)

`subprocess` call: check for execution of untrusted input

(S603)